examples/pymc3_howto/sampling_compound_step.ipynb

```diff
@@ -12,10 +12,24 @@
     "# Compound Steps in Sampling\n",
```

Comment:
<p>I would keep the original code because it has 2 advantages. 1st is that it shows how one can sample the same model as many times as they want with different samplers, iterations or whatever. 2nd is that this is an actual model with observed variables, the mu1, mu2 example only samples from free variables, it doen't sample from a posterior. We can still use coords and dims by using <code>idata_kwargs</code>  (in which case we don't need to use coords when defining the model)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/141/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>I tried keeping the original code and for some reason that didn't give me two values for "accept", so I was somewhat confused</p>

Reply:
I think the original code was not sampling with the custom steps defined, only printing them to show which order they define, so trace probably referred to the trace from the other model, sampled combining nuts and metropolis instead of combining two metropolis.

Reply:
in the latest commit, i sampled after defining steps

Reply:
Took a look at the code, it's probably due to BinaryGibbsMetropolis not having accept unline BinaryMetropolis, so there is no repeated name on which to concatenate the accept stat

Reply:
so I should go for binarymetropolis and metropolis instead of binarygibbs and metropolis right?

Reply:
I would use `BinaryMetropolis`. I am not sure how used are compound methods, but all the order and other explanations done with BinaryGibbsMetropolis will still be relevant, we will only be adding the repreated stats component

Reply:
yes, it worked like a charm, I'll push the latest commit, I think it looks almost ready now